runner.go

```diff
@@ -32,13 +33,22 @@ func init() {
 		os.Exit(2)
 	}
 	log.SetOutput(f)
+
+	config := &Config{
```

Comment:
the variables for host, db, etc would be a bonus, but really if you want then `-influx` to toggle these is fine